subprojects/workers/src/main/java/org/gradle/workers/internal/DefaultWorkerExecutor.java

```diff
@@ -183,16 +182,7 @@ private AsyncWorkCompletion submitWork(IsolatedParametersActionExecutionSpec<?>
     }
 
     private static String getWorkerDisplayName(Class<?> workActionClass, WorkParameters parameters) {
```

Comment:
Can we just remove this method entirely and inline it now?

Reply:
Yes, good catch.